### PR TITLE
Bug fixes. Should fix:

### PR DESCRIPTION
**Pull request by gravejester**
https://github.com/PowerShell/EditorSyntax/issues/9

https://github.com/PowerShell/EditorSyntax/issues/10

https://github.com/PowerShell/EditorSyntax/issues/11

### CODE REVIEW

**Comment by gerane** [comment]
Nice!

**Comment by gravejester** [comment]
@gerane Turned out two of the issues were related, so easy fix :)

Thanks for testing and finding these bugs btw!

**Comment by gerane** [comment]
No problem. I should just be able to copy the TM language into code to try these changes out shouldn't I? 

I use code as my daily driver, and I have always liked themes, so I will be paying closer attention for anything off.

**Comment by gravejester** [comment]
Yes, just copy the language file over the existing one, and restart code.

**Comment by daviwil** [comment]
Thanks man! I'll take a look at this later today when I'm back in Athens.

File: PowerShellSyntax.tmLanguage [hunk 1]
```diff
@@ -9,7 +9,7 @@
 		<string>psd1</string>
 	</array>
 	<key>name</key>
-	<string>PowerShell</string>
+	<string>Powershell</string>
```
Comment (daviwil) [inline PowerShellSyntax.tmLanguage]: Hmmm, was this necessary to get VS Code to operate correctly?  I wonder if it'd have a negative effect on other editors?

**Comment by gerane** [comment]
I haven't tried it yet. Got caught up in a bug in a vendor's product at
work. So I've wasted the last couple days dealing with foreign support I
can barely understand.

They blamed Microsoft like usual, even though I can't even begin to see how
it is related to Microsoft. I think that is just their go to answer.

File: PowerShellSyntax.tmLanguage (hunk 1 above)
Comment (gravejester) [inline PowerShellSyntax.tmLanguage]: Ah.. that was just a visual help for me when working with the definitions in Sublime Text, as I had two versions of the language files installed :) I'll revert it back.

**Comment by CodeCyclone** [comment]
I tested this out and it worked for me by updating "C:\Program Files (x86)\Microsoft VS Code\resources\app\extensions\powershell\syntaxes\PowershellSyntax.tmLanguage"

**Comment by daviwil** [comment]
Hey @gravejester, sorry for the delay on getting this merged.  Done now!

By the way, the VS Code team gave me write access to their repo so I should be able to approve your PRs there now ;)

**Comment by ftischhauser** [comment]
I'm still seeing some color coding issues even after replacing this file manually:

Example 1:
```
$email = "test@domain.com"
```

Example 2:
```
$XML = @"
<?xml version="1.0" encoding="UTF-16"?>
	<test>powershell.exe -Command "&amp; {if($var.'$name'){Write-Output 'hello world'}}"</test>
"@
Write-Output 'hello world'
```

Both work perfectly with VSCode 1.2.1 and vscode-powershell 0.6.1.

Thanks for looking into this :)

**Pull request merged**